x-pack/plugins/uptime/public/components/queries/monitor_charts/monitor_charts.tsx

```diff
@@ -98,6 +102,14 @@ export const MonitorCharts = ({
         }
       );
 
+      const checksDomain = upSeries.concat(downSeries).map(({ y }) => y);
```

Comment:
This could probably also use a plain english explanatory comment block

Reply:
Addressed in https://github.com/elastic/kibana/pull/28217/commits/93928941be97a60c5a504f2ee3883fd13d986609.